docs/book/src/topics/addons.md

```diff
@@ -168,6 +168,13 @@ Then install the Helm chart on the workload cluster:
 helm install --repo https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/repo cloud-provider-azure --generate-name --set infra.clusterName=${CLUSTER_NAME} --set "cloudControllerManager.clusterCIDR=${CCM_CIDR_BLOCK}"
 ```
 
+- **Note**: 
```

Comment:
Do we want to add some cross-reference in https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/docs/book/src/topics/flatcar.md as well?

Reply:
Good thought, will update!